src/main/resources/docker/Dockerfile

```diff
@@ -7,7 +7,10 @@ RUN groupadd -g 1000 -r sld \
  && useradd -u 1000 -g 1000 -r -m sld \
  && chmod +x /data/*.sh \
  && mkdir /var/lm \
- && chown sld:sld /var/lm /data
+ # set group to 0 to allow arbitrary users in OCP to write to this directory
+ && chown -R sld:0 /var/lm /data
```

Comment:
you should add here /home/sld too, otherwise with OCP and arbitrary users you will get a warning in the pod log file saying that the user can not access that folder.

Reply:
Ok, done.